src/plugin.js

```diff
@@ -41,6 +55,19 @@
     }
   }
 
+  Object.keys(backfillClasses).forEach(function(className) {
+    backfillClasses[className].forEach(function(name) {
+      var Component = videojs.getComponent(name);
+      var createEl = Component.prototype.createEl;
+
+      Component.prototype.createEl = function() {
```

Comment:
Is overriding createEl necessary for this? What about just adding the classes post-player init?